packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page/entity-type-tabs.tsx

```diff
@@ -4,13 +4,13 @@ import { Box, Tabs, tabsClasses } from "@mui/material";
 import { useRouter } from "next/router";
 import { useState } from "react";
 import { useFormContext, useWatch } from "react-hook-form";
-import { useFontLoadedCallback } from "../../../../components/hooks/useFontLoadedCallback";
-import { EntityTypeEditorForm } from "./form-types";
-import { TabLink } from "./tab-link";
-import { getTabUri, getTabValue, useCurrentTab } from "./use-current-tab";
-import { useEntityType } from "./use-entity-type";
-import { useEntityTypeEntities } from "./use-entity-type-entities";
-import { getEntityTypeBaseUri } from "./util";
+import { useFontLoadedCallback } from "../../../../../components/hooks/useFontLoadedCallback";
+import { useEntityTypeEntities } from "./shared/entity-type-entities-context";
+import { EntityTypeEditorForm } from "./shared/form-types";
+import { TabLink } from "./entity-type-tabs/tab-link";
+import { getEntityTypeBaseUri } from "./shared/get-entity-type-base-uri";
+import { getTabUri, getTabValue, useCurrentTab } from "./shared/tabs";
+import { useEntityType } from "./shared/entity-type-context";
```

Comment:
(optional)

```suggestion
import { useEntityType } from "./shared/entity-type";
```

The fact that it’s context is an implementation detail. As users of `useEntityType` we know that the value comes ‘from above’ but we don’t care if this is context, `react-query`, `redux`, Apollo cache or what not.